tests/test_registry.py

```diff
@@ -221,14 +235,33 @@ def test_clean_registries(self):
         self.assertEqual(self.testconn.zcard(started_job_registry.key), 0)
         self.assertEqual(self.testconn.zcard(failed_job_registry.key), 0)
 
+    def test_clean_registries_with_serializer(self):
+        """clean_registries() cleans Started and Finished job registries (with serializer)."""
+
+        queue = Queue(connection=self.testconn, serializer=JSONSerializer)
+
+        finished_job_registry = FinishedJobRegistry(connection=self.testconn, serializer=JSONSerializer)
+        self.testconn.zadd(finished_job_registry.key, {'foo': 1})
+
+        started_job_registry = StartedJobRegistry(connection=self.testconn, serializer=JSONSerializer)
+        self.testconn.zadd(started_job_registry.key, {'foo': 1})
+
+        failed_job_registry = FailedJobRegistry(connection=self.testconn, serializer=JSONSerializer)
+        self.testconn.zadd(failed_job_registry.key, {'foo': 1})
```

Comment:
Same thing here, do you mind adding test cases which would fail prior to this patch?

Reply:
@selwin This would have failed before, because the cleanup registry method on `BaseRegistry` calls `fetch` on the job class which in the past did not pass a serializer to that call, so this can stay.